src/ui/globals/constants.ts

```diff
@@ -3,9 +3,13 @@ const SEED_PHRASE_SUGGESTIONS = 3;
 const MAX_FAVOURITES = 5;
 const DOUBLE_TAP_DELAY = 300;
 
+// Links
 const DISCORD_LINK = "https://discord.com/invite/9yNneV8Ktv";
 const DOCUMENTATION_LINK =
```

Comment:
as a side note, do we have a ticket for updating this link when we have our documentation ready?

Reply:
No, we don't.

Reply:
thanks